GameData/RealismOverhaul/RO_DependentMods/RO_AJE.cfg

```diff
@@ -37,6 +37,177 @@
+
+	@MODULE[ModuleEngines*]
+	{
+		@name = ModuleEnginesAJEJet
+		%maxThrust = 27.8
+		%Area = 0.221	//Compressor Area. Guess
+		%BPR = 0		//Bypass Ratio
+		%CPR = 5.0		//Compressor Pressure Ratio, 4.0 effective
+		%FPR = 0		//Fan Ratio
+		%TPR = 0.9		//Turbine Pressure Ratio
+		%Mdes = 0.6		//Mach Design Point
+		%Tdes = 250		//Temp Design Point
+		%eta_c = 0.8	//Efficiency at burner inlet
+		%eta_t = 0.85	//Efficiency at burner exit
+		%eta_n = 0.7	//Efficiency at afterburner rear / nozzle entrance
+		%FHV = 30000000	//Fuel heat of burning (joules?)
+		%TIT = 1200		//Combustion peak temp
+		%TAB = 2000		//Afterburner temp?
+		%exhaustMixer = False
+		%thrustUpperLimit = 55
+		%maxT3 = 515	//Turbine max temperature
+		
+		// Engine fitting params
+		%dryThrust = 27.8
+		%wetThrust = 31
+		%drySFC = 1.14
+		
+		%flameoutEffectName = flameout
+		%powerEffectName = running_thrust
+		%engageEffectName = engage
+		%disengageEffectName = disengage
+		%spoolEffectName = running_turbine
+		%engineSpoolIdle = 0.05
+		%engineSpoolTime = 2.0
+		%EngineType = Turbine
+		
+		@PROPELLANT[LiquidFuel]
+		{
+			@name = Kerosene
+		}
+	}
+	!EFFECTS {}
+	EFFECTS
+	{		
+		running_thrust
+		{			
+			AUDIO
+			{
+				channel = Ship
+				clip = sound_jet_deep
+				volume = 0.0 0.0
+				volume = 0.05 0.4
+				volume = 1.0 1.0
+				pitch = 0.0 0.6
+				pitch = 1.0 1.0
+				loop = true
+			}
+			PREFAB_PARTICLE
+			{
+				prefabName = fx_smokeTrail_light
+				transformName = thrustTransform
+				emission = 0.0 0.0
+				emission = 0.05 0.0
+				emission = 0.075 0.25
+				emission = 1.0 1.25
+				speed = 0.0 0.25
+				speed = 1.0 1.0
+				localOffset = 0, 0, 1
+				localRotation = 1, 0, 0, -90
+			}					
+		}
+		running_turbine
+		{
+			AUDIO
+			{
+				channel = Ship
+				clip = sound_jet_low
+				volume = 0.0 0.0
+				volume = 0.05 0.7
+				volume = 1.0 1.0
+				pitch = 0.0 0.4
+				pitch = 0.05 0.5
+				pitch = 1.0 1.0
+				loop = true
+			}
+		}
+		engage
+		{
+			AUDIO
+			{
+				channel = Ship
+				clip = sound_vent_medium
+				volume = 1.0
+				pitch = 2.0
+				loop = false
+			}
+		}
+		disengage
+		{
+			AUDIO
+			{
+				channel = Ship
+				clip = sound_vent_soft
+				volume = 1.0
+				pitch = 2.0
+				loop = false
+			}
+		}
+		flameout
+		{
+			PREFAB_PARTICLE
+			{
+				prefabName = fx_exhaustSparks_flameout_2
+				transformName = thrustTransform
+				oneShot = true
+			}
+			AUDIO
+			{
+				channel = Ship
+				clip = sound_explosion_low
+				volume = 1.0
+				pitch = 2.0
+				loop = false
+			}
+		}
+	}
+}
+
 // Set RSSROConfig, set AvGas
 @PART[*]:HAS[@MODULE[ModuleEnginesAJEPropeller]]:FOR[RealismOverhaul]
 {
```

Comment:
This patch seems like it wants to be `AFTER[RealismOverhaul]` since it wants to modify any part that configured an AJE propeller.  I guess this is currently fine if AJE is the only mod providing that prop info, since your SXT patches are AJEJet instead of prop.

Reply:
It was like that when I found it